ts/packages/ts/src/ConfigSchema.ts

```diff
@@ -58,7 +58,7 @@ export abstract class ConfigSchema implements IConfigSchema {
         Currency: ({ value }) => validator.isISO4217(value),
         DockerImage: ({ value }) =>
           // eslint-disable-next-line no-useless-escape
-          /^(?:(?=[^:\/]{1,253})(?!-)[a-zA-Z0-9-]{1,63}(?<!-)(?:\.(?!-)[a-zA-Z0-9-]{1,63}(?<!-))*(?::[0-9]{1,5})?\/)?((?![._-])(?:[a-z0-9._-]*)(?<![._-])(?:\/(?![._-])[a-z0-9._-]*(?<![._-]))*)(?::(?![.-])[a-zA-Z0-9_.-]{1,128})?$/gm.test(
+          /^([\w.-]+\/)?[\w.-]+(?::[\w.-]+)?(\/[\w.-]+)*(?::[\w.-]+)?$/gm.test(
```

Comment:
This pattern allows uppercase letters which are illegal:

![image](https://github.com/configu/configu/assets/47181182/8508007b-b701-457a-a06e-d95389a48b35)